community/cypher/cypher-compiler-3.0/src/main/scala/org/neo4j/cypher/internal/compiler/v3_0/commands/expressions/Pow.scala

```diff
@@ -20,7 +20,7 @@
 package org.neo4j.cypher.internal.compiler.v3_0.commands.expressions
 
 case class Pow(a: Expression, b: Expression) extends Arithmetics(a, b) {
```

Comment:
Because Pow is not a function, but an expression, using the operator `^`. It neatly uses `TypeSafeMathSupport` instead, which I think is as nice as it extending `NullAndNaNSafeMathFunction`.